build/helper/documentation_snippets.py

```diff
@@ -96,5 +96,97 @@
 { 'simulate': True, 'driver_setup': { 'Model': '<model number>',  'BoardType': '<type>' } }
 '''
 
+default_close_function_doc = '''
+Closes the driver session and cleans up.
+'''
+
+close_function_note = '''
+This function is not needed when using the session context manager
```

Comment:
Do you think clients will understand what "the session context manager" is?
Honest question.

Reply:
By default we will use the documentation from the driver's close function if it exists. 

The note will be added regardless of whether the default description is used or the description from the driver. I am open on what the note should say to make clear that normally you don't need to use this function. I agree this isn't very good, but couldn't come up with a better way to say this.

Reply:
Looking at docs:
https://docs.python.org/3/whatsnew/2.6.html#pep-343-the-with-statement

They talk of the `with` statement, rather than of context managers.
I suggest following this terminology here as well as in initate/abort/initalize.

Let's refine that in its own PR so we don't hold this one.

Reply:
https://github.com/ni/nimi-python/issues/1020